packages/noya-renderer/src/components/EditablePath.tsx

```diff
@@ -51,6 +51,47 @@ function EditablePathPoint({ point, fill, stroke }: EditablePathPointProps) {
   );
 }
 
+interface EditableControlPointProps {
+  point: Point;
+  controlPoint: Point;
+  fill: Paint;
+  stroke: Paint;
+}
+
+function ControlPathPoint({
+  point,
+  controlPoint,
+  fill,
+  stroke,
+}: EditableControlPointProps) {
+  const { CanvasKit } = useReactCanvasKit();
+
+  const path = useMemo(() => {
+    const path = new CanvasKit.Path();
+
+    path.addRect(
+      CanvasKit.XYWHRect(
+        controlPoint.x - POINT_SIZE / 2 + 2,
```

Comment:
What does the `+ 2` do here?

Reply:
Moves the control point to center of the line. Without it, it looks like this: 
Open to alternative solutions :) 

<img width="465" alt="Screen Shot 2021-06-09 at 8 37 09 PM" src="https://user-images.githubusercontent.com/43770359/121460816-7ad9bc80-c962-11eb-9b5f-d5dd6e90101e.png">

Reply:
If we make another constant for this, we can use the same formula that we do with the main point:

```ts
const CONTROL_POINT_SIZE = 2

const rect = Primitives.rect(CanvasKit, {
  x: point.x - CONTROL_POINT_SIZE,
  y: point.y - CONTROL_POINT_SIZE,
  width: CONTROL_POINT_SIZE * 2,
  height: CONTROL_POINT_SIZE * 2,
});
```

^ Since the width/height are equal to size * 2, we can offset it by 50% by subtracting half the width/height from x/y

Reply:
oh good idea, constant has been added.